Example in README.md causes a compilation error

line 20 in the example (line 63 in the README.md) currently reads: 
`web.Middleware.create(web.middleware.LoggingMiddleware);`

But this causes a compilation error:
`example.zig:20:60: error: expected ',', found ';'`

To fix this compilation error  line 20 in the example should be changed to:
`web.Middleware.create(web.middleware.LoggingMiddleware),`

--- a/README.md
+++ b/README.md
@@ -49,8 +49,10 @@ pub const log_level = .info;
 
 const MainHandler = struct {
     pub fn get(self: *MainHandler, request: *web.Request, response: *web.Response) !void {
+        _ = self;
+        _ = request;
         try response.headers.put("Content-Type", "text/plain");
-        try response.stream.write("Hello, World!");
+        _ = try response.stream.write("Hello, World!");
     }
 
 };
@@ -60,13 +62,13 @@ pub const routes = [_]web.Route{
 };
 
 pub const middleware = [_]web.Middleware{
-    web.Middleware.create(web.middleware.LoggingMiddleware);
+    web.Middleware.create(web.middleware.LoggingMiddleware),
 };
 
 pub fn main() anyerror!void {
     var gpa = std.heap.GeneralPurposeAllocator(.{}){};
     defer std.debug.assert(!gpa.deinit());
-    const allocator = &gpa.allocator;
+    const allocator = gpa.allocator();
 
     var app = web.Application.init(allocator, .{.debug=true});
     defer app.deinit();

